404.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
Would it be possible to call this file `301.html` (for moved permanently) instead of `404.html` (not found)? Or would that break functionality entirely? If it must be `404.html`, does this mean a 404 is initially encountered when following links from this repo (followed by a 301 or 200)?

Reply:
This is the only way to make it work, as far as I know, with GitHub Pages. They don't support HTTP-level redirects like that.